platform/commonUI/browse/bundle.js

```diff
@@ -226,7 +232,7 @@ define([
                         "$window"
                     ],
                     "group": "windowing",
-                    "cssClass": "icon-new-window",
+                    "cssclass": "icon-new-window",
```

Comment:
Yea that was something that bugged me.  I don't know where or when it happened but I thought I manually changed it back.  Thanks!